src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt

```diff
@@ -27,16 +27,16 @@ import java.time.LocalDate
 @DiscriminatorColumn(name = "OWNER_CLASS")
 @Inheritance
 abstract class Address(
-  val premise: String? = null,
-  val street: String? = null,
-  val locality: String? = null,
+  open val premise: String? = null,
```

Comment:
All these need to open in this abstract class to allow Hibernate to create a proxy - this clears those warnings at startup